rosetta-source/src/main/rosetta/product-asset-enum.rosetta

```diff
@@ -132,3 +132,7 @@ enum FPVFinalPriceElectionFallbackEnum: <"Specifies the fallback provisions in r
 
 	FPVClose <"In respect of the Early Final Valuation Date, the provisions for FPV Close shall apply.">
 	FPVHedgeExecution <"In respect of the Early Final Valuation Date, the provisions for FPV Hedge Execution shall apply.">
+
+enum SpreadCalculationMethodEnum: <"Method by which asset swap spread is calculated, par-par or proceeds.">
```

Comment:
is there a definition of par-par or proceeds we can reference here? can we at least put them in quotation marks? "par-par" and "proceeds" 
Also is this only applicable to Asset Swaps or would we ever indicate a SpreadCalculationMethod for other products? Could this Enum expand in time to include other types of calculations? If so let's use asset swap as example. Consider description as follows:
<"Method by which a spread is calculated. E.g., "par-par" or "proceeds" are methods by which an asset swap spread is calculated typically.">

Reply:
I modified the description to be more generic.

The enum values which are derived from FpML model do not contain any description.

Reply:
@iansloyan can you please review again? Thank you!